core/cache/src/main/java/uk/gov/gchq/gaffer/cache/CacheServiceLoader.java

```diff
@@ -56,6 +56,12 @@ public static void initialise(final Properties properties) {
             }
             return;
         }
+
+        if (isEnabled()) {
```

Comment:
I think there needs to be a test added which fails before this change and passes to prove it works

Reply:
This was added because test were failing.